policyengine_us/parameters/gov/states/nj/tax/income/exemptions/dependents/amount.yaml

```
@@ -0,0 +1,11 @@
+description: NJ exemption amount per dependent child.
```

Doesn't look like it's only dependent children
```suggestion
description: New Jersey provides an exemption of this value per dependent.
```